hoomd/mpcd/geometry.py

```diff
@@ -124,6 +124,78 @@ def _attach_hook(self):
         super()._attach_hook()
 
 
+class RotatedParallelPlates(Geometry):
+    r"""Rotated parallel-plate channel.
+
+    Args:
+        separation (float): Distance between plates.
+        angle (float): Rotation angle in degrees.
```

Comment:
You might want to add a short paragraph below documenting that `angle` is not allowed to be 90 degrees. You can also mention that equivalent rotations can be obtained using values of `angle` between 0 and 45 degrees.